packages/rest-client/README.md

```diff
@@ -47,8 +47,8 @@ class ExampleApiClient extends RestClient {
     )
   }
 
-  async getExampleData(authOptions: AuthOptions) {
-    return this.get({ path: '/example-data' }, authOptions)
+  async getExampleData(username: string) {
```

Comment:
you don't then need the `AuthOptions` import above

Reply:
Oops, messed up my merge I think! Fixed that now :)